packages/markmap-cli/src/index.ts

```diff
@@ -144,6 +154,8 @@ export async function main() {
       if (offline) await fetchAssets();
       const content = await readFile(input, 'utf8');
       const output = cmd.output || `${input.replace(/\.\w*$/, '')}.html`;
+      const outputImage =
+        cmd.outputImage || `${input.replace(/\.\w*$/, '')}.png`;
```

Comment:
How about reuse `output` and only generate an image if `output` ends with `.png`?